src/index.js

```diff
@@ -1,18 +1,36 @@
-var express = require( 'express' );
-var app = express();
+const http = require( 'http' );
+
 const PORT = process.env.PORT || 3000;
+const BASEURL = process.env.BASEURL || 'http://localhost/';
 
-// Used by the monitoring system on VIP to verify the health of the app
-// Should return 200 when the app is healthy
-// https://docs.wpvip.com/technical-references/vip-platform/node-js/
-app.get( '/cache-healthcheck?', function( req, res ) {
-	res.sendStatus( 200 );
-} );
+const app = http.createServer( ( req, res ) => {
+	let { pathname: requestPath } = new URL( req.url, BASEURL );
+
+	// Remove trailing slash from the request path, if exists.
+	requestPath = ( requestPath.length > 1 && requestPath.endsWith( '/' ) ) ? requestPath.slice( 0, -1 ) : requestPath;
+
```

Comment:
> However, it is still trimming the trailing slash, if it exists. This helps, for example, to match both localhost:3000/example and localhost:3000/example/ to the same endpoint without the need for an additional condition.

I appreciate this approach and I think it makes sense in the context of a "real" application. Previously, the goal of this skeleton may have been to serve as a building block for such an application. However, we've seen that no one really uses this code. Instead they use a framework that implements routing in an opinionated way. Therefore, this is really just a generic example of how to implement a cache-healthcheck that keeps the site healthy until the customer commits their own code.

I'd remove this just to keep it simple. Note that we will never send `/cache-healthcheck/`, we always omit the trailing slash.

```suggestion
```

Reply:
Thanks, that makes perfect sense!